README.md

```diff
@@ -41,7 +41,7 @@ model = resnet50()
 nncf_config = NNCFConfig.from_json("resnet50_int8.json")
 
 # Provide data loaders for compression algorithm initialization, if necessary
-nncf_config = register_default_init_args(nncf_config, train_loader, loss_criterion)
+register_default_init_args(nncf_config, train_loader, loss_criterion)
```

Comment:
The return value is not redundant. We need to modify one of the arguments of a function, but it is better to do so explicitly by returning the modified value instead of relying upon pass-by-reference mechanism. I'd rather if you modified `register_default_init_args` so that it modifies a copy of the original config instead of the argument directly.

Reply:
agree as it's a library function

Reply:
But with an internal copy of nncf_config, users would have to change their code as well.
Otherwise, the sample config wouldn't have an updated nncf_config with registered structures.
```
config.nncf_config = register_default_init_args(nncf_config, train_loader, loss_criterion)
```
So kept as is, without copy and with returning the argument.